pandas/tests/indexing/multiindex/test_getitem.py

```diff
@@ -279,3 +280,32 @@ def test_loc_empty_multiindex():
     result = df
     expected = DataFrame([1, 2, 3, 4], index=index, columns=["value"])
     tm.assert_frame_equal(result, expected)
+
+
+@pytest.mark.parametrize("nan", [np.nan, pd.NA, None])
```

Comment:
There is a fixture for all NaNs

Reply:
nulls_fixture

Reply:
I fixed the test case to use nulls_fixture in the most recent commit